src/DssSpell.sol

```diff
@@ -16,197 +15,116 @@
 
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
-    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/6410a8f1f720099f7ba6553c84b9f9d966a3d319/governance/votes/Executive%20vote%20-%20September%203%2C%202021.md -q -O - 2>/dev/null)"
+    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/6d18889e1ebcc2f024c377e106682493ce399beb/governance/votes/Executive%20vote%20-%20September%2017%2C%202021.md -q -O - 2>/dev/null)"
     string public constant override description =
-        "2021-09-03 MakerDAO Executive Spell | Hash: 0xaa8b302da5c9a3f4bfd45685fd04398d3d35af1b4c1cbdfe83f166a6455de0ea";
-
-    address constant MCD_VEST_DAI = 0x2Cc583c0AaCDaC9e23CB601fDA8F1A0c56Cdcb71;
-    address constant MCD_VEST_MKR = 0x0fC8D4f2151453ca0cA56f07359049c8f07997Bd;
-
-    // Com Core Unit
-    address constant COM_WALLET     = 0x1eE3ECa7aEF17D1e74eD7C447CcBA61aC76aDbA9;
-    // Dai Foundation Core Unit
-    address constant DAIF_WALLET    = 0x34D8d61050Ef9D2B48Ab00e6dc8A8CA6581c5d63;
-    // Dai Foundation Core Unit (Emergency Fund)
-    address constant DAIF_EF_WALLET = 0x5F5c328732c9E52DfCb81067b8bA56459b33921f;
-    // GovAlpha Core Unit
-    address constant GOV_WALLET     = 0x01D26f8c5cC009868A4BF66E268c17B057fF7A73;
-    // Growth Core Unit
-    address constant GRO_WALLET     = 0x7800C137A645c07132886539217ce192b9F0528e;
-    // Marketing Content Production Core Unit
-    address constant MKT_WALLET     = 0xDCAF2C84e1154c8DdD3203880e5db965bfF09B60;
-    // Oracles Core Unit
-    address constant ORA_WALLET     = 0x2d09B7b95f3F312ba6dDfB77bA6971786c5b50Cf;
-    // Protocol Engineering
-    address constant PE_WALLET      = 0xe2c16c308b843eD02B09156388Cb240cEd58C01c;
-    // Risk Core Unit
-    address constant RISK_WALLET    = 0xd98ef20520048a35EdA9A202137847A62120d2d9;
-    // Real-World Finance Core Unit
-    address constant RWF_WALLET     = 0x9e1585d9CA64243CE43D42f7dD7333190F66Ca09;
-    // Ses Core Unit
-    address constant SES_WALLET     = 0x87AcDD9208f73bFc9207e1f6F0fDE906bcA95cc6;
-
-    uint256 constant MAY_01_2021 = 1619827200;
-    uint256 constant JUN_21_2021 = 1624233600;
-    uint256 constant JUL_01_2021 = 1625097600;
-    uint256 constant SEP_01_2021 = 1630454400;
-    uint256 constant SEP_13_2021 = 1631491200;
-    uint256 constant SEP_20_2021 = 1632096000;
-    uint256 constant OCT_01_2021 = 1633046400;
-    uint256 constant NOV_01_2021 = 1635724800;
-    uint256 constant JAN_01_2022 = 1640995200;
-    uint256 constant MAY_01_2022 = 1651363200;
-    uint256 constant JUL_01_2022 = 1656633600;
-    uint256 constant SEP_01_2022 = 1661990400;
-
-    uint256 constant MILLION = 10 ** 6;
-    uint256 constant WAD     = 10 ** 18;
-
-    // Turn off office hours
+        "2021-09-17 MakerDAO Executive Spell | Hash: 0x613a50aee82adca6f6f32a4f7298c70b1c9d061648e786c84b3872ffc5963e3e";
+
```

Comment:
Meant to add this tribute to an older spell.  If we can add it in this one, I would be grateful.

```suggestion

    string public constant in_memory_of = "Jeffrey Blechschmidt";
    
```